.github/workflows/test_on_push.yml

```diff
@@ -95,28 +91,25 @@ jobs:
 
       - name: Run unit tests for GNU/Linux with Python 3.9 and generate coverage report
         if: matrix.os == 'ubuntu-latest' && matrix.python-version == 3.9
-        run: tox -e coverage
+        run: |
+          python -m tox -e coverage
 
       - name: Upload coverage report
         if: matrix.os == 'ubuntu-latest' && matrix.python-version == 3.9
         uses: codecov/codecov-action@v2.1.0
 
-      - name: Run integration tests for GNU/Linux
-        if: matrix.os == 'ubuntu-latest'
+      - name: Run integration tests for GNU/Linux with Python 3.11
+        if: matrix.os == 'ubuntu-latest' && matrix.python-version == 3.11
```

Comment:
This is right now failing while installing SUNDIALS and SuiteSparse. But, I think this will fail even if the SUNDIALS step passes because of `jax` and `jaxlib`. We might have to make some changes to `tox.ini` to ensure that `pybamm_install_jax` runs only on `3.8` and `3.9`.

`JaxSolver` is currently not being tested on Python `3.8` as well, but it should be. If you look closely -
- `[testenv:coverage]` installs `jax`, making it available while executing `tox -e coverage`
- The base `[testenv]` only installs `jax` on `tests-!windows-!mac`, which isn't used on CI anywhere, rather it should install `jax` in `integration` and `unit` but only on Python `3.8` and `3.9`.

I'll look into the `jax` and `jaxlib` thing.

Reply:
That sounds right, we are indeed missing 3.8 with `JaxSolver` – though I'll leave this over to you like you ask :)

Reply:
I think the previous run wasn't failing due to jaxlib but due to some error with `wget` internally, SUNDIALS and SuiteSparse were installed successfully on this one

Reply:
Sure, I'll add a note about `JaxSolver` in the docs for installing PyBaMM from source. Please let me know if I need to add it anywhere else too